index.html

```diff
@@ -34,48 +50,47 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
     </div>
       <ul class="iconns"> 
         <li>
-          <a href="" class="icon"> <img src="images/icons/twitter.svg" alt=""></a>  
+          <a href="" class="icon"> <img src="images/icons/twitter.svg" alt="twitter"></a>  
         </li>  
          <li>
-          <a href="" class="icon"> <img src="images/icons/i.png" alt="">
-            <img src="images/icons/n.png" alt="">
+          <a href="" class="icon"> <img src="images/icons/i.png" alt="linked in ">
+            <img src="images/icons/n.png" alt=" linked in">
            </a> 
          </li>
         <li>
-          <a href="" class="icon">  <img  class="medium" src="images/icons/medium@2x.png" alt=""></a>
+          <a href="" class="icon">  <img  class="medium" src="images/icons/medium@2x.png" alt=" medium"></a>
         </li>
         <li>
-          <a href="" class="icon"><img src="images/icons/github.svg" alt=""></a>
+          <a href="" class="icon"><img src="images/icons/github.svg" alt=" github"></a>
         </li>
         <li>
-          <a href="" class="icon"><img src="images/icons/angellist.svg" alt=""></a>
+          <a href="" class="icon"><img src="images/icons/angellist.svg" alt=" angellist"></a>
         </li>
          
       </ul>
       <div class="scroll">
-        <img src="images/Normal Button/Disabled.png" alt="">
+        <img src="images/Normal Button/Disabled.png" alt=" scroll-button">
       </div>
   </section>
-  <section class="all-works">
+  <section class="all-works" id="portfolio">
   
-      <div class="recent-works"><h3 class="hey-text">My Recent Works</h3>
+      <div class="recent-works"><h2 class="hey-text">My Recent Works</h2>
       </div>
 <div class="green"></div>
-      <div class="project-cards">
+      <div class="T">
         <!--Project section-1 -->
         
-        <div class="project-section-1 project-1">
+        <div class="project-section-1 project1">
           <div class="card"></div>
-        
-          <div class="project-work">
-            <h4 class="title-1">Multi-Post Stories<br>Gain+Glory</h4>
+          <div class="project-work ">
+            <h3 class="title-1">Multi-Post Stories<br>Gain+Glory</h3>
             <ul class="list">
               <li class="ruby">Ruby on rails</li>
               <li class="css">css</li>
               <li class="javascript">JavaScript</li>
```

Comment:
- [ ] Also, I can see that popup is not responsive. It is only looking good in desktop. But in laptop or mobile version, It's completely broken. Please fix this.

![Screenshot from 2023-03-12 03-29-33](https://user-images.githubusercontent.com/26767383/224513285-3dc442ce-2fee-4835-987d-e36b595222be.png)